platform/iphone/app_delegate.mm

```diff
@@ -29,15 +29,14 @@
 /*************************************************************************/
 
 #import "app_delegate.h"
+#import "gl_view.h"
```

Comment:
`#import` and `#include` are kind of the same in Obj-C, no? Would be good to homogenize it and use either `#import` or `#include` but be consistent with it. Then it should also follow the style guide, so:
```
#import "app_delegate.h"

#import "core/project_settings.h"
#import "drivers/coreaudio/audio_driver_coreaudio.h"
#import "gl_view.h"
...

#import <GameController/GameController.h>
```

Reply:
Since this is a `.mm` file, I think we can agree it's Objective-C++, so all bets are off 😉 

The `GameController` framework header doesn't have `#ifdef` guards, since `#import` prevents them. However `#include` a framework header may now result in duplicate symbols.

In this case I'm leaning towards all `#import`

Reply:
Not sure if another opinion is wanted, but I think it should be #import for ObjC(++) headers, #include for C/C++ headers. #import could obfuscate circular include issues in C/C++ headers if they were included in non-ObjC targets.